test/test_pysocks.py

```diff
@@ -112,6 +111,7 @@ def assert_proxy_response(self, resp_data, content, address,
                          self.test_server.request['headers']['user-agent'])
         self.assertEqual('%s:%d' % address,
                          self.test_server.request['headers']['host'])
+        content = b'3\r\n' + content + b'\r\n0'
```

Comment:
I havent investigated why the 3 & 0 are appearing.

Also I noticed on openSUSE_Factory_zSystems' arch 's390x' this fails with
> AssertionError: '3\r\nzzz\r\n0' != '3\r\nzzz\r\n'